### DISCUSSION TITLE
Colorblind Mode Makes Traffic Insight Same Color

### DISCUSSION THREAD

**Discussion by ds5678** [community/community#9117] [Accessibility] [answered]
The colorblind mode is great for issues and pull requests, but it makes the Traffic Insight graphs use the same color for both lines.

**Comment by gaknoll** [comment]
Thanks for sharing this feedback @ds5678! We are working on a new release for the colorblind theme now and I'll share this with the team 🙏🏽

**Comment by Juliusschaeper** [comment]
Thanks @ds5678, would you mind attaching a screenshot or link for reference?

**Reply by ds5678** [reply]
Normal
![Screenshot 2022-02-04 121330](https://user-images.githubusercontent.com/49847914/152572889-7bd805b3-36e4-4c38-9ed3-e26256e21a10.png)
Colorblind
![Screenshot 2022-02-04 121426](https://user-images.githubusercontent.com/49847914/152572937-2ccb97f3-e6c7-4354-9ba3-5fc305444d53.png)

**Reply by ds5678** [reply]
Any update on this?

**Reply by ericwbailey** [reply, marked as answer]
Hi there, @ds5678. I'm a member of the GitHub accessibility team, and I've been tasked with addressing this issue.

We just released an update to hopefully address the issue you reported. Here are some screenshots of the before and after treatment when a colorblind theme is active:

#### Before

![A Traffic chart showing two lines for clones and uniques. The color of the two lines is almost identical. Additionally there is a tooltip showing total numbers that also uses a color key. Both colors in the key look nearly identical](https://github.com/user-attachments/assets/d0a20ae7-172c-48c1-9191-ba0d75a2fee6)

#### After

![The traffic chart updated to show two distinct color values: blue and pink. Additionally, the points on the graph for the uniques have been updated to use small squares instead of circles.](https://github.com/user-attachments/assets/9d343000-6428-4a9d-8122-aacafc578464)

You may need to refresh your browser tab for these changes to take effect.

Additionally, I wanted to thank you for the reminder, and for your patience while we worked on a fix. I hope this change helps you better use the data visualization.

**Reply by ds5678** [reply]
Thank you!

**Comment by cehfisher** [comment]
Hello @ds5678 - thank you for providing this feedback! We can still see that this is an issue and will file this with our Accessibility team so that it can be reviewed. We will follow-up here with any additional questions or updates. Thanks again!

**Comment by 0349664307** [comment]
0349664307
